job-server/test/spark.jobserver/WebApiMainRoutesSpec.scala

```diff
@@ -352,6 +352,9 @@ class WebApiMainRoutesSpec extends WebApiSpec {
       Post("/contexts/meme?num-cpu-cores=3&coarse-mesos-mode=true") ~> sealRoute(routes) ~> check {
         status should be (OK)
       }
+      Post("/contexts/meme?num-cpu-cores=3", "spark.context-settings.test=1") ~> sealRoute(routes) ~> check {
```

Comment:
After understanding the flow a bit more, this is another approach to testing this: https://github.com/casey-green/spark-jobserver/compare/casey/context-spark-config-in-body...casey-green:casey/context-spark-config-in-body-alt-test-approach?expand=1